### PR TITLE
Testing the house without a proper painting

### PR DESCRIPTION
**Pull request by dsonne47**
Can you please review this house

### CODE REVIEW

File: house.txt [hunk 1]
```diff
@@ -1,13 +1,13 @@
     ______||__
    ///////////\
-  ///////////__\
- ///////////____\
+  ///////////  \
+ ///////////    \
  |    _    |     |
- |[] | | []| []  |-----------------||
```
Comment (dsonne47) [inline house.txt:6]: fencing removed.

Comment (dsonne47) [reply house.txt:6]: @VaibhavDubey1  can you check this

File: house.txt [hunk 2]
```diff
@@ -1,13 +1,13 @@
     ______||__
    ///////////\
-  ///////////__\
- ///////////____\
+  ///////////  \
+ ///////////    \
  |    _    |     |
```
Comment (VaibhavDubey1) [inline house.txt:5]: @dsonne47 : Why are u making this change ?

File: house.txt [hunk 3]
```diff
@@ -1,13 +1,13 @@
...
- |[] | | []| []  |-----------------||
- |   | |___|_____|        _____    ||
-     \_______  _|_ \    _/[] []\   ||
-             \  _|_ \  '-o----o-'  ||
-	   		  \  _|_ \_____________||
```
Comment (VaibhavDubey1) [inline house.txt:10]: @dsonne47 : This car is appearing distorted.

Comment (VaibhavDubey1) [inline house.txt:10]: Can u see my comments?

**Review by dsonne47 — COMMENTED** [review]
Waiting for approval

File: house.txt (hunk 2 above)
Comment (dsonne47) [reply house.txt:5]: just for fun

File: house.txt (hunk 1 above)
Comment (dsonne47) [reply house.txt:6]: yeah i am checking on it

**Review by dsonne47 — COMMENTED** [review]
I am done

**Pull request merged**